### PR TITLE
Add entrypoints to packages with internal exports

### PR DESCRIPTION
**Pull request by scottn12**
## Description

Add Named Exports for Release Tags to Packages with internal only exports

Follow up from https://github.com/microsoft/FluidFramework/pull/18899

### CODE REVIEW

File: examples/apps/attributable-map/package.json [hunk 1]
```diff
@@ -11,6 +11,24 @@
...
+		},
+		"./internal": {
+			"import": {
+				"types": "./lib/index.d.mts",
+				"default": "./lib/index.mjs"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.cjs"
+			}
```
Comment (tylerbutler) [inline examples/apps/attributable-map/package.json:29]: In general you want to have an import and require section in each export except in cases where the package is only building one module type.

Comment (scottn12) [reply examples/apps/attributable-map/package.json:29]: How can I tell if there is only one module type?

File: packages/tools/fetch-tool/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"main": "dist/fluidFetch.js",
+	"main": "dist/index.js",
```
Comment (tylerbutler) [inline packages/tools/fetch-tool/package.json:13]: Are you sure abou this one?

Comment (scottn12) [reply packages/tools/fetch-tool/package.json:13]: oops missed this, not sure how that happened

**Comment by tylerbutler** [comment]
Replaced by https://github.com/microsoft/FluidFramework/pull/18977 since I can't push to this PR branch for some reason.

**Pull request closed without merging**